src/pages-and-resources/discussions/app-config-form/apps/legacy/LegacyConfigForm.jsx

```diff
@@ -36,8 +37,8 @@ function LegacyConfigForm({
     <Card className="mb-5 pt-3 px-5 pb-5">
       <Form ref={formRef} onSubmit={handleSubmit}>
         <h3>{title}</h3>
-        <AppConfigFormDivider />
-        <DivisionByGroupFields
+        <AppConfigFormDivider thick />
+        <InContextDiscussionFields
```

Comment:
`InContextDiscussionFields` can't be here included in `LegacyConfigForm` - this is the legacy form and doesn't support in-context discussions.  I don't think the Figma files are correct in this specific case; I think they're showing the "new" discussions fields.